src/main/java/eu/europa/ted/efx/sdk1/EfxTemplateTranslatorV1.java

```diff
@@ -39,6 +41,8 @@
 public class EfxTemplateTranslatorV1 extends EfxExpressionTranslatorV1
     implements EfxTemplateTranslator {
 
+  private static final Logger logger = LoggerFactory.getLogger(EfxTemplateTranslator08.class);
```

Comment:
private static final Logger logger = LoggerFactory.getLogger(EfxTemplateTranslator08.class); 08 should be V1

Reply:
Side effect of the merge with develop. It sneaked past the previous PR's review.
It's fixed now.